lib/Backend/JITTimePolymorphicInlineCacheInfo.cpp

```diff
@@ -53,8 +53,9 @@ JITTimePolymorphicInlineCacheInfo::InitializePolymorphicInlineCacheInfo(
     __in Js::PolymorphicInlineCacheInfo * runtimeInfo,
     __out PolymorphicInlineCacheInfoIDL * jitInfo)
 {
+#pragma warning(suppress: 6001)
```

Comment:
inside operator=, there's *this, preFast though it's dereferencing and using an uninitialized memory, here itself is actually assignment(i.e. initializing)
before the change polymorphicCacheUtilizationArray is not wrapped with WriteBarrierPtr and does not have operator= and *this

---
In reply to: [92454774](https://github.com/Microsoft/ChakraCore/pull/2191#discussion_r92454774) [](ancestors = 92454774)